s3torchbenchmarking/utils/prepare_ec2_instance.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Script to prepare an EC2 instance for PyTorch benchmarks.
+
+set -euo pipefail
+
+# Sanity check
+sudo yum -y upgrade # OR, `sudo apt -y upgrade`
+
+# Activate the default PyTorch env
+#
+# This command only works on specific AMI (DL-AMI): make sure to check the compatibility between the latter and the EC2
+# instance type currently in use (if incompatible, the command will either fail or generate a warning).
+source activate pytorch
+
+# Install Rust (https://www.rust-lang.org/learn/get-started)
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
why do we need rust if we install s3torchconnectorclient from pip?

Reply:
I recall needing that to build the client, but I believe I was installing it using `-e`, in which case Rust is required... will remove.